api/client/network.go

```diff
@@ -113,6 +113,7 @@ func (cli *DockerCli) CmdNetworkRm(args ...string) error {
 			status = 1
 			continue
 		}
+		fmt.Fprintf(cli.out, "%s\n", net)
```

Comment:
Yes, that is true. While what I am thinking about is to keep consistency of all Docker CLI's code style. Almost all the CLI side code uses `Fprintf`.